examples/lighting-app/python/BUILD.gn

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("${chip_root}/build/chip/tools.gni")
+import("${chip_root}/src/app/common_flags.gni")
+
+assert(chip_build_tools)
+
+import("${chip_root}/examples/common/pigweed/pigweed_rpcs.gni")
+
+if (chip_enable_pw_rpc) {
+  import("//build_overrides/pigweed.gni")
+  import("$dir_pw_build/target_types.gni")
+}
+
+config("includes") {
+  include_dirs = [
+    ".",
+    "include",
+    "${chip_root}/zzz_generated/lighting-app/",
+  ]
+}
+
+shared_library("chip-lighting-app") {
+  sources = [
+    #    "${chip_root}/src/controller/python/chip/setup_payload/Generator.cpp",
```

Comment:
please do not keep commented-out code, delete it instead.

This looks like an empty shared library - is it needed?

Reply:
You're correct. This was a remainder from the earliest tests on where to build the library. Complete file removed.